### PR TITLE
Lab6기반으로 Lab11 DropOut 예제 작성

### PR DESCRIPTION
**Pull request by HyoyoungRainy**
lab6_runTFMultiANN_spiraldata.py 작성 기준
error rate를 줄이기 위해
Learning rate
Activation function (sigmoid → tanh)
Training epoch를 수정
Lab6 코드를 기반으로 dropout 코드작성
lab11_runTFTanhDropOut_spiraldata.py 작성
( tf.layers.dropout  추가)
충분한 dropout 효과를 보기위해 Lab6에 비해 hidden node의 개수를 각 레이어별로 두배 찍 증가

### CODE REVIEW

**Comment by HyoyoungRainy** [comment]
please refer to the document 
https://docs.google.com/document/d/1aaBpB5lHiglllEKo04f6jehFHfrTlebx21ME09wc_5o/edit
to observe the effect of DropOut (which I observed)
